src/Lazy.ts

```diff
@@ -305,3 +305,6 @@ lazyImageFieldValue();
 
 import { lazyQuerySuggestPreview } from './ui/QuerySuggestPreview/LazyQuerySuggestPreview';
 lazyQuerySuggestPreview();
+
+import { lazyQueryForCommerce } from './ui/QueryForCommerce/LazyQueryForCommerce';
```

Comment:
Is there a reason not to add the new component in the Eager.ts file as well?

Reply:
Just that I wasn't aware at that moment :). Done.